### PR TITLE
Fix issue https://github.com/cloudfoundry/bosh/issues/2236

### PR DESCRIPTION
**Pull request by gu-bin**
This is to fix issue https://github.com/cloudfoundry/bosh/issues/2236

### CODE REVIEW

**Comment by xtreme-conor-nosal** [comment]
Hi @gu-bin 

There are also unit tests for the ERB templating. Please update the tests and ensure `bundle exec rake spec:release_unit` passes

**Comment by xtreme-conor-nosal** [comment]
Might also want to add the domain to the local_dns feature in the /info response for consistency https://github.com/cloudfoundry/bosh/blob/master/src/bosh-director/lib/bosh/director/api/controllers/info_controller.rb#L26-L32

**Comment by h4xnoodle** [comment]
@gu-bin we'll be able to merge this once the CLA passes, there are tests, and the recommended addition. Thanks.

**Comment by gu-bin** [comment]
@xtreme-conor-nosal @h4xnoodle I've addressed all the comments. Please review again. Thanks.

**Comment by gu-bin** [comment]
Anything else do I need to do to get this PR merged?

**Comment by gu-bin** [comment]
@xtreme-conor-nosal @h4xnoodle Please review again. This time I didn't modify the original powerdns case but only add `.else` for the bosh-dns case. Any other problem please let me know. Thanks.

**Comment by gu-bin** [comment]
@xtreme-conor-nosal @h4xnoodle Any comments for this?

**Comment by gu-bin** [comment]
@xtreme-conor-nosal @h4xnoodle Any update for this PR? We need it to be fixed. Thanks.

File: src/bosh-director/lib/bosh/director/api/controllers/info_controller.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ class InfoController < BaseController
       def initialize(config)
         super(config)
         @powerdns_manager = PowerDnsManagerProvider.create
+        @localdns_manager = LocalDnsManager.create(Config.root_domain, @logger)
```
Comment (mrosecrance) [inline src/bosh-director/lib/bosh/director/api/controllers/info_controller.rb:9]: This call seems unnecessary as you've just accessed the root_domain below directly

**Comment by mrosecrance** [comment]
Both people you've requested for review have rotated off the team, so I apologize for the delay getting back to you.

File: src/bosh-director/lib/bosh/director/api/controllers/info_controller.rb (hunk 1 above)
Comment (gu-bin) [reply src/bosh-director/lib/bosh/director/api/controllers/info_controller.rb]: I've removed this line.

**Comment by gu-bin** [comment]
@mrosecrance I've addressed your comments. Anything else do I need to do before it can be merged?

**Pull request merged**